src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -15,25 +16,31 @@ public class ArmSubsystem extends EntechSubsystem{
 
   private CANSparkMax elbowMotor;
   private CANSparkMax telescopeMotor;
-
+  private boolean enabled = false;
+  
   public ArmStatus getStatus(){
       return new ArmStatus();
   }
   
   @Override
   public void initialize() {
-    elbowMotor = new CANSparkMax(RobotConstants.ARM.ELBOW_MOTOR_ID, MotorType.kBrushed);
-    telescopeMotor = new CANSparkMax(RobotConstants.ARM.TELESCOPE_MOTOR_ID, MotorType.kBrushed);
+	if ( Robot.isReal() ) {
```

Comment:
I think we need to comment out anything motor related in order not to crash on the test base because the motors aren't found therefore the elbowMotor and telescopeMotor are nulls.  John had to completely comment out the initialize() and periodic() methods to stop null references

Reply:
ok i'll add an enable flag that disables it all.